keyboards/ebastler/isometria_75/rev1/halconf.h

```diff
@@ -0,0 +1,525 @@
+/*
```

Comment:
Don't need all of this. 

Just need something like this: 

https://github.com/qmk/qmk_firmware/blob/01c7108db52677c2a4210bcda0afd2b1d3f7619f/platforms/chibios/keyboard-config-templates/halconf.h